ros_bt_py/ros_bt_py/tree_node.py

```diff
@@ -430,12 +431,9 @@ def main(argv=None):
         executor = SingleThreadedExecutor()
         executor.add_node(tree_node)
 
-        try:
-            executor.spin()
-        finally:
-            executor.shutdown()
-            tree_node.destroy_node()
```

Comment:
Are these shutdowns what did the masking?

Reply:
The finally clause did the masking. I removed it in favor of catching the KeyboardInterruption